README.md

```diff
@@ -176,6 +176,70 @@ Releases and publishes the package. Runs tests, lints and builds the project bef
 
 This command uses [`standard-version`](https://github.com/conventional-changelog/standard-version) underneath. The version is automatically inferred from the [conventional commits](https://conventionalcommits.org/).
 
+## Using a linked version of [@discussify-styleguide](https://github.com/ipfs-shipyard/discussify-styleguide)
+
+In some cases, you may want to make changes to Discussify's styleguide at the same time as you work on your project which uses the styleguide. In order to use a local version of [@discussify-styleguide](https://github.com/ipfs-shipyard/discussify-styleguide) and have any styleguide modifications be reflected live on your project, some pages have to be made in our main project.
+
+### Install `postcss-import-webpack-resolver`
+
+Run the following command in your main project
+
+```sh
+$ npm i postcss-import-webpack-resolver
+```
+
+### Update `postcss-preset-moxy` to the latest version
```

Comment:
I would remove this heading and merge it in the previous heading because they are both installations necessary. The reason is that you don’t really need to install postcss-preset-moxy to consume the styleguide, unless you want to use its variables.

Reply:
I will remove the instruction to install `postcss-preset-moxy` as it is not necessary to pass the custom resolver.

Reply:
Hmm but perhaps it would be better to mention postcss-preset-moxy? The reason is that is already mentioned in the regular installation, in case you want to use variables or Nixon’s.

Reply:
Maybe there should be a note stating that if using `postcss-preset-moxy`, the dependency should be updated to a version equal or older than 3.0.0 and that the import attribute should be in the object passed as an argument to the function imported with `postcss-preset-moxy`.

Reply:
Yep.